README.md

```diff
@@ -7,24 +7,86 @@
+  <a target=_blank>https://docs.substrate.io/</a>
+</h3>
 
-> Always create your own branch to work on the site. Use `develop` for testing. Both Netlify flags towards the bottom display status of `develop` and `main` site. `main` is production ONLY and deployed by dedicated Dev or Comms team through Forestry.
+## Contributing
 
-### Status
+Thank you for your interest in contributing to documentation for the Substrate development
+framework. As a member of the community, you are invited and encouraged to contribute by submitting
+issues, offering suggestions for improvements to existing content, adding review comments to
+existing pull requests, proposing new content, or creating new pull requests to fix issues or
+provide new content. Please review our [contributor guidelines](/v3/contribute/style-guide) prior to
+any contribution. If you have any further questions, please do not hesitate to reach out on our 
+[substrate technical community channel](https://matrix.to/#/#substrate-technical:matrix.org)! We
+would love to get to know you and your work!
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/65f522fe-eefa-434b-bdb3-6345d363c177/deploy-status)](https://app.netlify.com/sites/ecstatic-babbage-c109a3/deploys) ![linkcheck workflow](https://github.com/substrate-developer-hub/substrate-docs/actions/workflows/check-links.yml/badge.svg)
+### Directory Structure
+
+The content of this website is versioned in separate directories `v<version>` is the convention
+used.The devhub is then hierarchically separated into the three main types of documentation: docs,
+how-to-guides, and tutorials. 
+
+Sub folders in these are of the form:
+```
+- <version>
+  - <doc type>
+    - <XX-section>
+      - <Y-page>
+        - `index.mdx`
+```
+By convention we use `XX` numbering starting at `00` for sections, and `Y` lettering starting at `a`
+for pages. 
+
+File directory example: `v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx`
+
+Each `index.mdx` page has the content to be rendered to this page, and all pages include a header
+section with a `slug` item that is used for navigation on the generated site, 
 
-[Staging Site](https://substrate-docs-staging.netlify.app)
+URL example: https://docs.substrate.io/v3/contribute/style-guide
+
+Configuration and styling files for gatsby live primarily in the `src` folder.
+
+Some gotchas:
+- The `i18n` folder is used for translation **but also for the rendering of the navigation**
+  **elements sections**. Your `index.mdx` file's `title` is not the source for this.
+- If you are adding or renaming a page, you **must** add it correctly in `src/components/DevNavMenu.tsx` and
+  possibly `gatsby-config.js` and `gatsby-node.js`.
+
+### Production Deployment
+
+- `main` branch is available at: https://docs.substrate.io/
+
+### Staging Deployment
+
+- `develop` branch is available at: https://substrate-docs-staging.netlify.app
+
+<!-- internal note:
+***Important***
```

Comment:
this should move into the style guide, but i didn't want to touch that file ;) we can open a new PR for that.